AtEventDisplay/AtSidebar/AtSidebarIntMacro.cxx

```diff
@@ -0,0 +1,18 @@
+#include "AtSidebarIntMacro.h"
+
+#include <FairLogger.h> // for endl, basic_ostream, cout, ostream
```

Comment:
/user/anthonya/attpcroot/AtEventDisplay/AtSidebar/AtSidebarIntMacro.cxx should remove these lines:
- #include <FairLogger.h>  // lines 3-3